test/commands/job/sasjs-job.spec.js

```diff
@@ -0,0 +1,69 @@
+import { saveGlobalRcFile } from '../../../src/utils/config-utils'
+import dotenv from 'dotenv'
+import path from 'path'
+import { processJob } from '../../../src/sasjs-job/index'
+
+describe('sasjs context', () => {
+  beforeAll(async () => {
+    await saveGlobalRcFile(
+      JSON.stringify({
+        targets: [
+          {
+            name: 'cli-tests',
+            serverType: 'SASVIYA',
+            serverUrl: 'https://sas.analytium.co.uk',
+            appLoc: '/Public/app/cli-tests',
+            tgtServices: ['testJob'],
+            tgtDeployVars: {
+              contextName: 'SharedCompute'
```

Comment:
The SharedCompute context doesn't exist in our environment any more - perhaps we can use the SAS Job Execution Context by default?  That will always exist